recipe/meta.yaml

```diff
@@ -67,6 +73,16 @@ requirements:
     - ffmpeg
     - fontconfig
     - collada-dom
+    - libxcb                             # [linux]
+    - xcb-util-wm                        # [linux]
+    - xcb-util-keysyms                   # [linux]
+    - xcb-util-image                     # [linux]
+    - xcb-util-renderutil                # [linux]
+    - xorg-libx11                        # [linux]
+    - xorg-libxext                       # [linux]
+    - xorg-libsm                         # [linux]
+    - xorg-libice                        # [linux]
+    - xorg-xf86vidmodeproto              # [linux]
```

Comment:
most of these will be added automatically as a run export. did things fail for you want to add them?

Reply:
No, I realize that afterward...

Reply:
I let the CI finish to see if it works, I have a doubt on se linux

Reply:
got it. the second challenge is to remove the yum requirements too. similar "guideline" for that one.

Reply:
OK:
1) I remember why i needed libselinux the first time, it was not throwing a compilation error without it, but the app `openscenegraph` failed launching otherwise, so I think i should keep it, I have no option to test it without merge
2) the package I put in run requirements are exactly the same as the one in qt-main repository

So is this recipe ok for you ? I will keep migrating from cdt to package for xcb and x11 in the downstream package.

Is it ok for you ?